doc/python_reference.rst

```diff
@@ -534,6 +534,14 @@ Events
    combine_event_ids
    equalize_epoch_counts
 
+Classes:
+
+.. autosummary::
+   :toctree: generated/
+   :template: class.rst
+
+   ElektaAverager
```

Comment:
Actually, the correct place would be line 31 ``io.ElektaAverager``. Sorry for the noise.